src/styles/blocks/head-block/icon.scss

```diff
@@ -0,0 +1,7 @@
+.icon {
```

Comment:
Icon class properties are not defined using BEM methodology. BEM methodology recommends that elements should be named based on the block they are nested in. For example, if the icon is within a block named 'header', then the icon should be named 'header__icon'.